R/init.R

```diff
@@ -155,8 +155,8 @@ init <- function(data,
                  modules,
                  title = NULL,
                  filter = list(),
-                 header = tags$p("Add Title Here"),
-                 footer = tags$p("Add Footer Here"),
```

Comment:
It makes sense! Is there a way to add these interactively afterwards?

Reply:
You mean after the app starts? I don't think so. They can be specified by the app developer.

Reply:
Ok, if someone wants to do that they can ask for it as a new feature probably. For the moment, it was not used imo

Reply:
it is not checked though (no assertion). Is it only because it is a shiny.tag?

Reply:
There were no assertions previously. I will add them, good idea :+1: